kde-apps/kmail/kmail-9999.ebuild

```diff
@@ -25,7 +25,8 @@ RESTRICT="test" # bug 616878
 COMMON_DEPEND="
 	>=app-crypt/gpgme-1.23.1-r1:=[cxx,qt6]
 	dev-libs/ktextaddons:6[speech?]
-	>=dev-qt/qtbase-${QTMIN}:6[dbus,gui,network,widgets]
+	dev-libs/qtkeychain[qt6]
```

Comment:
`dev-libs/qtkeychain-0.14.1-r1` introduced IUSE qt6, which means this is the minimum version you should require here.